source/data/landings.yaml

```diff
@@ -687,6 +699,10 @@
     title: 'Front End Performance Guide'
     subtitle: "Ready to dive into Pantheon's Global CDN? Check out this guide to learn how to ace an online speed test."
     url: '/guides/frontend-performance'
+  cta_alt:
+    title: 'Website Performance with Varnish, Redis, and New Relic'
+    subtitle: 'How does Pantheon help your website be faster than other platforms?'
```

Comment:
Minor thing: Maybe 'How does Pantheon help your website run faster than other platforms?'

Reply:
I like it, added. (but forgot to mention it in the commit)